themes/book/app/routes/$.tsx

```diff
@@ -150,24 +154,24 @@ export default function Page() {
     ...pageDesign,
   };
   return (
-    <ArticlePageAndNavigation
-      hide_toc={hide_toc}
-      hideSearch={hide_search}
-      projectSlug={data.page.project}
-    >
-      {/* <ProjectProvider project={project}> */}
-      <ProjectProvider>
-        <ComputeOptionsProvider
-          features={{ notebookCompute: true, figureCompute: true, launchBinder: false }}
-        >
-          <ThebeLoaderAndServer baseurl={baseurl}>
+    <ProjectProvider>
+      <ComputeOptionsProvider
+        features={{ notebookCompute: true, figureCompute: true, launchBinder: false }}
+      >
+        <ThebeLoaderAndServer baseurl={baseurl}>
```

Comment:
Lifted these up so that the footer behaves similarly to an article page w.r.t computation.